2d/benchmarks/dambreak_Colagrossi/helpers.py

```diff
@@ -1,12 +1,14 @@
 def CreateFig():
     from tables import  openFile
-    archive = openFile('dambreak_Colagrossi0.h5','r')
+    archive = openFile('dambreak_Colagrossi_p.h5','r')
     import dambreak_Colagrossi
     import dambreak_Colagrossi_so
     import matplotlib.tri as mtri
     from matplotlib import pyplot as  plt
     import numpy as np
     domain = dambreak_Colagrossi.domain
+    domain.L = dambreak_Colagrossi.tank_dim
```

Comment:
@cekees , they can be obtained with the `getBoundingBox` function of Domain: https://github.com/erdc-cm/proteus/blob/master/proteus/Domain.py#L68-L79 and then calling `domain.L` and `domain.x`